includes/elements/section.php

```diff
@@ -418,16 +418,26 @@ protected function _register_controls() {
 		$this->add_control(
 			'content_position',
 			[
-				'label' => __( 'Content Position', 'elementor' ),
+				'label' => __( 'Vertical Align', 'elementor' ),
```

Comment:
Why?

Reply:
Shilo asked for it. 
In column settings, the same control is labeled Vertical Align, I assume he wanted uniformity.
I think this change was agreed upon a while ago and no one got around to doing it.